src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -140,6 +156,16 @@ export default {
 			// remove any text contained within square brackets, including the brackets
 			return String(this.name).replace(/\s\[.*\]/g, '');
 		},
+		monthlyGoodPromoData() {
```

Comment:
Without the categoryName prop, you can do this same switch logic using:
`this.name` or `this.id` ( I added the id in my commit for GROW-189